queues/queue_queries.sh

```diff
@@ -0,0 +1,12 @@
+# >> START cli-queue-queries A set of Genesys Cloud CLI Queuries for queues
+
+#Query all of the queues, sort them by memberCount and name, pull the fields we want with JQ and the dump them out to a CSV file format (you can just dump the JSON), but it is not always usable.
+gc queues list --pageSize 100 | jq -r '. | sort_by(.memberCount,.name)| reverse' | jq -r '. | map({id: .id, name: .name, membercount: .memberCount})' | jq -r '(.[0] | keys_unsorted) as $keys | ([$keys] + map([.[ $keys[] ]])) [] | @csv' 
```

Comment:
Lines don't wrap in code blocks displayed in the dev center, so using some line wraps in long commands like these may be beneficial to readability.